docs/loader_usage.md

```diff
@@ -0,0 +1,114 @@
+# Scylla Cluster Tests (SCT): Loaders and Stress Commands
+
+This document explains how to configure and use loaders and stress commands effectively in Scylla Cluster Tests (SCT). It primarily focuses on Longevity Tests (`longevity_test.LongevityTest.test_custom_time`), although the concepts apply similarly to other tests, which might include additional parameters (refer to specific test code/documentation for details).
+
+---
+
+## Overview of Loaders
+
+Loaders are specialized machines used to generate workloads on Scylla clusters. They run stress tools such as:
+
+- `cassandra-stress`
+- `scylla-bench`
+- `gemini`
+- `latte`
+- `cql-stress`
+- `ycsb`
+
+Loaders execute these stress tools within Docker containers (Docker images specified in `defaults/docker_images`). They help simulate realistic database loads and measure performance and reliability.
+
+---
+
+## How Loaders Operate
+
+The stress load is initiated usually via methods such as:
+
+- `sdcm.tester.ClusterTester.run_stress_thread`
+- `sdcm.utils.loader_utils.LoaderUtilsMixin.assemble_and_run_all_stress_cmd`
+
+### Stress Command Parameters
+
+Stress commands are specified using these parameters:
+
+- **`stress_cmd`**: Used primarily in longevity tests.
+- **`stress_cmd_<w|r|m>`**: Used in performance tests, with `<w|r|m>` representing write, read, or mixed workloads.
+
+These parameters accept either a single string or a list of strings, each representing a distinct stress command.
+
+### Adjustments to Stress Commands
+
+Stress commands provided by the user undergo the following modifications before execution:
+
+- **Node targeting**: IP addresses of all nodes in the cluster are appended automatically.
+- **Duration adjustment**: Modified according to the test’s `stress_duration` and `prepare_stress_duration` parameters, if present.
+- **HDR Histogram**: if `use_hdrhistogram` is set to `true`, proper arguments are added and SCT handles collection HDR histograms.
```

Comment:
I know that not all tools support the HDR Histogram. probably the same for other adjustments.  Mybe we need some "support matrix" here?

Reply:
c-s 
latte
ycsb - WIP
scylla-bench - in theory / wasn't used in SCT - we check will it out for gocql based perf test 

others no one cares, at least so far.

Reply:
added